src/core/presence.ts

```diff
@@ -147,71 +137,66 @@ export interface Presence extends EmitsDiscontinuities {
 
   /**
    * Subscribe the given listener from the given list of events.
+   *
+   * Note that this method will throw an error if presence events are not enabled in the room options.
+   * Make sure to set `enableEvents: true` in your room options to use this feature.
+   *
    * @param eventOrEvents {'enter' | 'leave' | 'update' | 'present'} single event name or array of events to subscribe to
    * @param listener listener to subscribe
+   * @throws An {@link Ably.ErrorInfo} with code 40000 if presence events are not enabled
    */
   subscribe(eventOrEvents: PresenceEvents | PresenceEvents[], listener?: PresenceListener): Subscription;
 
   /**
    * Subscribe the given listener to all presence events.
+   *
+   * Note that this method will throw an error if presence events are not enabled in the room options.
+   * Make sure to set `enableEvents: true` in your room options to use this feature.
```

Comment:
Same as for occupancy, mention it's under `presence` somehow. Maybe add a `@link` for both to the config setting?